tracer/build/_build/PrepareRelease/SetAllVersions.cs

```diff
@@ -281,6 +281,10 @@ public override void Run()
                     "src/Datadog.Trace.Tools.Runner/Datadog.Trace.Tools.Runner.csproj",
                     NugetVersionReplace);
 
+                SynchronizeVersion(
+                    "src/Datadog.Trace.Trimming/Datadog.Trace.Trimming.csproj",
+                    NugetVersionReplace);
```

Comment:
You should add this file to the list of "updated" files in the `VerifyChangedFilesFromVersionBump`

Reply:
Done in [0617805](https://github.com/DataDog/dd-trace-dotnet/pull/4246/commits/06178058379e730505e9c839e6ef4254c8bcf5be)